src/test/perl/rbe_value_format.t

```diff
@@ -140,35 +140,64 @@ Readonly my @TEST_ARRAY => ('confFile', 'logFile', 'unused', 'logKeep', 'logFile
 Readonly my $FORMATTED_ARRAY => 'confFile logFile unused logKeep logFile';
 Readonly my $FORMATTED_ARRAY_SORTED => 'confFile logFile logFile logKeep unused';
 Readonly my $FORMATTED_ARRAY_UNIQUE => 'confFile logFile logKeep unused';
-my $rbe_fh = CAF::RuleBasedEditor->open($FILENAME, log => $obj);
+# When LINE_VALUE_OPT_SINGLE is set, formatAttributeValue() does nothihng and returns the array
+Readonly my $FORMATTED_ARRAY_SINGLE => 'ARRAY\(0x[a-f\d]+\)';
```

Comment:
how is this a "feature" that is unittested? should return undef because it tried to convert impossible type